lib/capsule_crm/taggable.rb

```diff
@@ -3,9 +3,11 @@ module Taggable
     extend ActiveSupport::Concern
 
     def tags
-      CapsuleCRM::Connection.get(
+      collection = CapsuleCRM::Connection.get(
         "/api/#{api_singular_name}/#{id}/tag"
-      )['tags']['tag'].map { |item| CapsuleCRM::Tag.new(item) }
+      )
+      collection['tags']['@size'] == '1' ? tags = [collection['tags']['tag']] : tags = collection['tags']['tag']
```

Comment:
Perhaps this could be fixed with coercion?:

```ruby
tags = CapsuleCRM::Connection.get(
  "/api/#{api_singular_name}/#{id}/tag"
)['tags']['tag']
Array(tags).map { |item| CapsuleCRM::Tag.new(item) }
```